pycee/errors.py

```diff
@@ -241,7 +244,7 @@ def handle_type_error(error_message):
     message = ""
     if hint1 in error_message:
         message = "must have first callable argument"
-    elif hint2 in message:
+    elif hint2 in error_message:
         message = remove_exception_from_error_message(error_message)
 
```

Comment:
@LeonardoFurtado, add ``message = slugify(message, separator="+")`` here

Reply:
that's perfect, you only have to reinsert  the return command ``return url_for_error(message)`` as this is still needed